lib/plugins/aws/package/compile/events/sqs/index.js

```diff
@@ -19,8 +19,9 @@ class AwsCompileSQSEvents {
           type: 'object',
           properties: {
             arn: { $ref: '#/definitions/awsArn' },
-            batchSize: { type: 'integer', minimum: 1, maximum: 10 },
+            batchSize: { type: 'integer', minimum: 1, maximum: 10000 },
             enabled: { type: 'boolean' },
+            maximumBatchingWindowInSeconds: { type: 'integer', minimum: 0 },
```

Comment:
Sorry, for second call. I just realized we should actually call it `maximumBatchingWindow`, to be on pair with spec proposed for new events here https://github.com/serverless/serverless/issues/8137

Reply:
No worries!
I've changed it: https://github.com/serverless/serverless/commit/d70c58b49c8a30ec194102801727f1b570db37d0